empty-project/src/server/app.ts

```diff
@@ -0,0 +1,4 @@
+import { startApp } from 'modelence/server';
+
+startApp({
```

Comment:
Minor: might be nice to include an empty array as a starter since adding a new module will be a requirement for pretty much any project and this will make it easier to do it correctly and in the right place

```suggestion
startApp({
  modules: [/* Add your modules here */]
```